### ISSUE TITLE
`functional/functional-parameters` in recommended/lite configs not working anymore

### ISSUE THREAD

**Issue by essenmitsosse** [eslint-functional/eslint-plugin-functional#885]
## Bug Report

After the change in #811 that changed the options for `functional/functional-parameters` in recommended and lite configs the rule is not working anymore. It is still working though when defined manually via `functional/functional-parameters: "error"`.

### Expected behavior

Those functions should throw an error for `functional/functional-parameters`.

```ts
function add() {
    return arguments.reduce((sum, number) => sum + number, 0);
}

const foo = () => 3;
```

### Actual behavior

They don't throw an error if the rule is used via the recommended or lite configs.

### Steps to reproduce

Use the recommended or lite config and write any function without arguments.

### Proposed changes

Either I am not understanding something about the changes in #811 (and maybe also #803) — in which case the documentation should be updated, or the new configuration for the rule in the recommended and lite configs is not working anymore as intended.
